### DISCUSSION TITLE
Deploy to App Store and Google Store

### DISCUSSION THREAD

**Discussion by VasilchenkoSergei** [shadowwalker/next-pwa#171] [Q&A]
I have React SPA with Next.js. I need to turn it into an app to deploy to two main Stores. I think I'll make it with PWA. So here's a question - Is there any problems to deploy SSR app (React/Next.js) to App Store and Google Store?

**Comment by KnlnKS** [comment]
You cannot deploy to the App Store. However, you can deploy to the Play Store quite painlessly using the Bubblewrap cli.
